Packs/CommunityCommonScripts/Scripts/ToCase/ToCase.yml

```
@@ -0,0 +1,34 @@
+args:
+- defaultValue: capital
+  description: Default is 'capital'. Can also be 'title' or 'swap'.
+  name: case
+  auto: PREDEFINED
+  predefined:
+  - capital
+  - title
+  - swap
+  required: true
+- default: true
+  name: value
+commonfields:
+  id: toCase
+  version: -1
+contentitemexportablefields:
+  contentitemfields:
+    fromServerVersion: ''
+dockerimage: demisto/python3:3.12.7.117934
```

```suggestion
dockerimage: demisto/python3:3.11.10.115186
```
Please update the Docker image to the following due to a current technical issue.